Tasks/SshV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -14,6 +14,8 @@
   "loc.input.help.scriptPath": "Path to the shell script file to run on the remote machine.",
   "loc.input.label.inline": "Inline Script",
   "loc.input.help.inline": "Write the shell script to run on the remote machine.",
+  "loc.input.label.interpreterCommand": "Interpreter command",
+  "loc.input.help.interpreterCommand": "Path to the command interpreter used to execute the script.",
```

Comment:
Could you please describe it more - what is the interpreterCommand here and how it will be added to the script?